kbi/0016/index.rst

```diff
@@ -18,10 +18,26 @@ could be that only specific files should be dropped, for example all files that
 to the dataset by a specific commit, while all other files that are available locally
 should remain untouched.
 
-This Knowledge Base Item outlines two methods for dropping local files that were added in
-a specific commit. These methods differ in the way they identify which files to drop (one
-via ``datalad status`` and one via ``git diff``), but the actual dropping of content is
-handled in the same way.
+This Knowledge Base Item outlines several methods for dropping local files that were added
+in a specific commit. These methods differ in the way they identify which files to drop
+(via ``datalad status``, ``git diff``, or ``datalad diff``), but the actual dropping of
+content is handled by ``datalad drop`` in all cases.
+
+Content
+-------
+
+- `Preparation`_
+- `Using datalad status`_
+- `Using git diff`_
+- `Dropping the files`_
+- `Using datalad diff in a one-liner`_
+- `Drop limitation`_
+
+*Note: if you are not interested in details and just looking for the quickest and leanest
+way to get the job done, skip over to the section: `Using datalad diff in a one-liner`_*
```

Comment:
```suggestion
.. note::
   If you are not interested in details and just looking for the quickest and leanest
   way to get the job done, skip over to the section: `Using datalad diff in a one-liner`_*.
```
The link didn't render for me - I don't know how to make inline emphasis and hyperlink markup play nicely together, but maybe a note directive would look nicely?